com.itopia.client.json

```diff
@@ -0,0 +1,245 @@
+                      "type": "git",
+                      "url": "https://salsa.debian.org/rousseau/PCSC.git",
+                      "tag": "pcsc-1.8.26"
+                  }
+              ]
+          },
+          {
+              "name": "ffmpeg",
+              "buildsystem": "autotools",
+              "config-opts": [
+                  "--enable-shared",
+                  "--enable-swresample",
+                  "--disable-static",
+                  "--disable-avdevice",
+                  "--disable-avcodec",
+                  "--disable-avformat",
+                  "--disable-postproc",
+                  "--disable-avfilter",
+                  "--disable-programs",
+                  "--disable-doc"
+              ],
+              "cleanup": [
+                  "/share/ffmpeg"
+              ],
+              "sources": [
+                  {
+                      "type": "git",
+                      "url": "https://github.com/FFmpeg/ffmpeg.git",
+                      "tag": "n4.2.1"
+                  }
+              ]
+          },
+          {
+              "name": "openh264",
+              "buildsystem": "simple",
+              "build-commands": [ "make -j PREFIX=/app install" ],
+              "sources": [
+                  {
+                      "type": "git",
+                      "url": "https://github.com/cisco/openh264.git",
+                      "tag": "v1.8.0"
+                  }
+              ]
+          },
+          {
+              "name": "freerdp",
+              "buildsystem": "cmake-ninja",
+              "config-opts": [
+                  "-DCMAKE_INSTALL_LIBDIR:PATH=lib",
+                  "-DCHANNEL_TSMF:BOOL=OFF",
+                  "-DCHANNEL_URBDRC:BOOL=ON",
+                  "-DCHANNEL_PRINTER:BOOL=ON",
+                  "-DCHANNEL_PRINTER_CLIENT:BOOL=ON",
+                  "-DBUILD_TESTING:BOOL=OFF",
+                  "-DWITH_MANPAGES:BOOL=OFF",
+                  "-DWITH_GSSAPI:BOOL=OFF",
+                  "-DWITH_PCSC:BOOL=ON",
+                  "-DWITH_SERVER:BOOL=OFF",
+                  "-DWITH_SAMPLE:BOOL=OFF",
+                  "-DWITH_CUPS:BOOL=OFF",
+                  "-DWITH_OPENH264:BOOL=ON",
+                  "-DWITH_OPENH264_LOADING:BOOL=OFF",
+                  "-DWITH_FFMPEG:BOOL=OFF",
+                  "-DWITH_DSP_FFMPEG:BOOL=OFF",
+                  "-DWITH_OSS:BOOL=OFF",
+                  "-DWITH_PULSE:BOOL=ON",
+                  "-DWITH_ALSA:BOOL=OFF",
+                  "-DWITH_OSS:BOOL=OFF",
+                  "-DWITH_CAIRO:BOOL=ON",
+                  "-DWITH_SWSCALE:BOOL=ON",
+                  "-DWITH_WINPR_TOOLS:BOOL=OFF",
+                  "-DWITH_WAYLAND:BOOL=OFF",
+                  "-DWITH_CLIENT:BOOL=OFF"
+              ],
+              "sources": [
+                  {
+                      "type": "git",
+                      "url": "https://github.com/FreeRDP/FreeRDP.git",
+                      "branch": "master"
```

Comment:
@reisbel you do want to change this to `"tag":  "2.0.0"` to have a specific version build. (you might need to update this periodically if we release new FreeRDP versions)

Reply:
@akallabeth yes please, go ahead and apply the change.